annazer.gemspec

```diff
@@ -2,15 +2,25 @@
 
 Gem::Specification.new do |s|
   s.name        = 'annazer'
-  s.version     = '0.0.1'
+  s.version     = '0.0.2'
   s.date        = '2019-09-16'
   s.summary     = 'Configuration gem for RoR applications'
   s.description = 'A centralized code quality guidelines gem for all Ruby on Rails projects'
   s.authors     = ['Omar Abubakr', 'Yara Salah']
   s.email       = 'omar.abubakr@espace.com.eg'
-  s.files       = ['lib/annazer.rb', 'lib/annazer/railtie.rb', 'lib/tasks/generate_config_files.rake', 'lib/annazer/config_files/.rubocop.yml', 'lib/annazer/config_files/.overcommit.yml']
+  s.files       = ['lib/annazer.rb', 'lib/annazer/railtie.rb', 'lib/tasks/generate_config_files.rake',
```

Comment:
You can use this line instead
```
  spec.files         = Dir.chdir(File.expand_path(__dir__)) do
    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
  end
```
The `git ls-files -z` loads the files in the RubyGem that have been added into git.